app-crypt/libu2f_server/libu2f_server-1.1.0.recipe

```diff
@@ -0,0 +1,86 @@
+SUMMARY="Yubico universal 2nd factor (U2F) server C library"
+DESCRIPTION="libu2fhost is a server-side implementation \
+of the U2F protocol.
+It provides an API for generating the JSON blobs that are
+required for the U2F Registration and U2F Authentication
+operations. Moreover, it provides functionality for the
+verification of the cryptographic operations."
```

Comment:
Check the 80 character limit for the lines in DESCRIPTION, keep paragraphs together with a backslash at the end of a line, move new paragraphs to  new line.